modules/RTC/RTC.js

```diff
@@ -433,12 +434,22 @@ export default class RTC extends Listenable {
      * @return {TraceablePeerConnection}
      */
     createPeerConnection(signaling, iceConfig, isP2P, options) {
+        const pcConstraints = RTC.getPCConstraints(isP2P);
+
+        if (typeof options.forceSuspendVideo !== 'undefined') {
```

Comment:
Will forceSuspendVideo be used outside of AB testing? Right now it's not clear by name that this is an ab test, but maybe that's intentional?

(The AB testing frameworks I'm used to will also have a separate ab testing helper, which makes it more clear what blocks to remove once the test is over. If this is an AB test, maybe it's worthwhile to consider having all checks go through a helper instead of passing in more options.)

Reply:
> Will forceSuspendVideo be used outside of AB testing?

No, its only use is this AB test.

Reply:
Renamed